File: Solution/Backend/dotnet/DataHelperClasslib_Generic/CSVFileIo/Entities/SkillRequirement.cs
namespace CSVFileIO.Entity
{
    public class SkillRequirement
    {
        public string Skill { get; set; }

        public string RequiredLevel { get; set; }

        public int CandidatesCovered { get; set; }

        public int TotalRequired { get; set; }

        public int FillPercentage { get; set; }

        public override string ToString()
        {
            return "\"" + Skill + "\"," +
                   RequiredLevel + "," +
                   CandidatesCovered + "," +
                   TotalRequired + "," +
                   FillPercentage;
        }
    }
}

File: Solution/Backend/java/webapi/src/main/java/com/transflower/tflcomentor/fileio/Entity/SkillRequirement.java
package com.transflower.tflcomentor.fileio.Entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class SkillRequirement {
    private String skill;
    private String requiredLevel;
    private int candidatesCovered;
    private int totalRequired;
    private int fillPercentage;
}
